src/poliastro/tests/tests_twobody/test_perturbations.py

```diff
@@ -31,3 +32,35 @@ def test_J2_propagation_Earth():
 
     assert_quantity_allclose(raan_variation_rate, -0.172 * u.deg / u.h, rtol=1e-2)
     assert_quantity_allclose(argp_variation_rate, 0.282 * u.deg / u.h, rtol=1e-2)
+
+
+def test_atmospheric_drag():
+    # http://farside.ph.utexas.edu/teaching/celestial/Celestialhtml/node94.html#sair (10.148)
+    # given the expression for \dot{a} / a, aproximate \Delta a \approx F_a * \Delta t
+
+    R = Earth.R.to(u.km).value
+    k = Earth.k.to(u.km**3 / u.s**2).value
+
+    # parameters of a circular orbit with h = 10 km
+    h = 250  # km
+    r0 = np.array([R + h, 0, 0])  # km
+    v0 = np.array([0, np.sqrt(k / (R + h)), 0])  # km/s
+
+    # parameters of a body
+    C_D = 2.2  # dimentionless
+    A = ((np.pi / 4.0) * (u.m**2)).to(u.km**2).value  # km^2
+    m = 100  # kg
+    B = C_D * A / m
+
+    # parameters of the atmosphere
+    rho0 = Earth.rho0.to(u.kg / u.km**3).value  # kg/km^3
+    H0 = Earth.H0.to(u.km).value
+
+    orbit = Orbit.from_vectors(Earth, r0 * u.km, v0 * u.km / u.s)
+    tof = 100000  # s
+
+    dr_expected = -B * tof * rho0 * np.exp(-(norm(r0) - R) / H0) * np.sqrt(k * norm(r0))
```

Comment:
This is related to the approximation trick you mention in the first comment I guess. Can we move that comment next to this line, and put `* tof` at the end?